### ISSUE TITLE
FreeBSD is not supported by this extension

### ISSUE THREAD

**Issue by MikeJakubik** [EnterpriseDB/system_stats#37]
[root@fbsd14 /tmp/system_stats]# PATH="/usr/local/pgsql/bin:$PATH" gmake USE_PGXS=1
"Platform is: freebsd"
Makefile:12: *** FreeBSD is not supported by this extension.  Stop.

Booo!.. Why not, what's the issue? Non portable C code?

**Comment by dpage-edb** [comment]
Yes, this extension has, by its nature, a significant amount of platform specific code.

PRs are welcome of course, but it's unlikely any of the existing devs will add FreeBSD support.
